Added course name features

bennettliu opened the pull request:
Added text fields in the options menu for entering and displaying course names to the bell schedule

bennettliu committed:
Added course name features

Added text fields in the options menu for entering and displaying course names to the bell schedule

bennettliu committed:
Consistently use options[]

Consistently use options[] instead of directly using localStorage.

bokken12 commented on scripts/script.js:
```diff
@@ -1146,6 +1149,27 @@ function attachOptionActions() {
     document.getElementsByName("color")[0].addEventListener("change", function (event) {
         updateSchedule(null, true);
     });
+    document.getElementsByName("period1")[0].addEventListener("change", function (event) {
```
Can we use a loop in this section rather than have repetitive code?

bokken12 reviewed (changes requested):
Ryan did a pretty good job reviewing, but I'll try to add a few things.

I don't think the github repository link should scroll, it's typically in the bottom right corner and in small screens it can now move up somewhat awkwardly
![](https://cdn.discordapp.com/attachments/379677812887519234/536651009582170122/Screen_Shot_2019-01-20_at_12.58.57_PM.png)

This feels a little much on a small phone screen
![](https://cdn.discordapp.com/attachments/379677812887519234/536655719345160240/image0.png)
I wonder if something more like this might be reasonable (or some other compromise)
![](https://cdn.discordapp.com/attachments/379677812887519234/536655862396092456/image0.png)
(and maybe edit the CSS such that the class name text box uses the same gray background as the notification display time box).

bennettliu committed:
Added Joel's proposed changes

Added Joel's proposed changes, namely: Made period text boxes smaller and gray. Fixed the Github repository link to the bottom of the scroll. Used a loop to add listeners to period text boxes.
